File: internal/content/testdata/bench/java/set-union.java
import java.util.*;

class Solution {
    public long[] set_union(long[] a, long[] b) {
        Set<Long> seen = new TreeSet<>();
        for (long x : a) seen.add(x);
        for (long x : b) seen.add(x);
        long[] result = new long[seen.size()];
        int i = 0;
        for (long x : seen) result[i++] = x;
        return result;
    }
}

File: internal/content/testdata/bench/csharp/set-union.cs
using System.Collections.Generic;
using System.Linq;

public class Solution {
    public long[] set_union(long[] a, long[] b) {
        var seen = new HashSet<long>(a);
        foreach (long x in b) seen.Add(x);
        var result = seen.ToList();
        result.Sort();
        return result.ToArray();
    }
}
